command/server.go

```diff
@@ -1751,8 +1751,10 @@ func (c *ServerCommand) Run(args []string) int {
 					err = pprof.Lookup(dump).WriteTo(pFile, 0)
 					if err != nil {
 						c.logger.Error("error generating pprof data", "name", dump, "error", err)
+						pFile.Close()
 						break
 					}
+					pFile.Close()
```

Comment:
Can `defer pFile.Close()` be used here instead of calling twice?

Reply:
This is in a for loop and doesn't return.